src/administrative-sdk/organisation/organisation-controller.js

```diff
@@ -16,7 +16,7 @@ export default class OrganisationController {
   }
 
   /**
-   * Create an organisation.
+   * Create an organisation. The organisation will be owned by the current active tenant.
```

Comment:
I meant **currently active** instead of **current active**. I'm not 100% sure though.